lighthouse-core/gather/gather-runner.js

```diff
@@ -216,6 +216,28 @@ class GatherRunner {
     return new LHError(LHError.errors.CHROME_INTERSTITIAL_ERROR);
   }
 
+  /**
+   * Returns an error if we try to load a non-HTML page.
+   * @param {LH.Artifacts.NetworkRequest|undefined} mainRecord
+   * @return {LH.LighthouseError|undefined}
+   */
+  static getNonHtmlError(mainRecord) {
+    // MIME types are case-insenstive
```

Comment:
there's no test for the case insensitivity

does chrome perhaps normalize this anyhow? you can hack the `static-server.js` to return any mime type you want and run `yarn static-server` to verify.

Reply:
yeah I couldn't find information online as to whether chrome automatically makes mimetypes lowercase regardless of the http response, i'll check with the method you mentioned and let you know

Reply:
ok update so I replaced all instances of 'text/html' with 'TEXT/HTML' in the static-server Content-Type header responses & confirmed in devtools that the Content-Type header showed up as 'TEXT/HTML'. When I ran a lighthouse run against a page hosted on static-server, I got no error and the mimeType accessible from mainRecord was always lowercase (text/html), so it seems that when chrome automatically makes its mimeType lowercase! 

So I am going to remove the case insensitivity modifier on the RegEx, which also makes me wonder if it should instead just be a direct string comparison with a const 'text/html' since thats likely more efficient and seems sufficient based on the test above

Reply:
nice! yeah a direct string comparison would be nice here. to be clear, the performance was never an issue (it's negligible), just the desire to keep things as complex as necessary (aka as simple as possible).

Reply:
a comment that chrome normalizes the mime type would be good too

Reply:
sounds good! i'll make sure to add the comment right now